doc/library-user.rst

```diff
@@ -72,6 +72,9 @@ handled as follows:
 2. If the fixed issue is non-critical, it will be delayed until the next major
    release (following the normal Neptune guarantees).
 
+Also note that a breaking patch release will be merged into subsequent minor
+releases, which *will not* be explicitly marked as containing breaking changes.
```

Comment:
Maybe also stress the point that any subsequent releases won't be marked either, despite being technically breaking when you upgrade from older version skipping some?

Reply:
I intended "subsequent minor releases" to cover this. You don't think that's clear enough?

Reply:
It also applies to subsequent patch releases.

Reply:
Maybe also mention something like "because of that, one should always pay more attention to release announcements with `+breaking` even if there is no immediate intention to upgrade - to know what problems can be expected eventually"

Reply:
Updated (also added second commit with more info).